client/src/leap/soledad/client/crypto.py

```diff
@@ -38,100 +37,51 @@
 MAC_KEY_LENGTH = 64
 
 
-def _assert_known_encryption_method(method):
+def encrypt_sym(data, key):
     """
-    Assert that we can encrypt/decrypt the given C{method}
-
-    :param method: The encryption method to assert.
-    :type method: str
-
-    :raise UnknownEncryptionMethodError: Raised when C{method} is unknown.
-    """
-    valid_methods = [
-        crypto.EncryptionMethods.AES_256_CTR,
-        crypto.EncryptionMethods.XSALSA20,
-    ]
-    try:
-        soledad_assert(method in valid_methods)
-    except AssertionError:
-        raise crypto.UnknownEncryptionMethodError
-
-
-def encrypt_sym(data, key, method):
-    """
-    Encrypt C{data} using a {password}.
-
-    Currently, the only encryption methods supported are AES-256 in CTR
-    mode and XSalsa20.
+    Encrypt C{data} using a {password} withAES-256 in CTR.
```

Comment:
Maybe fix it like this?
`Encrypt data using a key with AES-256 cipher in CTR mode.`